eul/Components/RamProcessRowView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  RamProcessRowView.swift
+//  eul
+//
+//  Created by Jevon Mao on 1/18/21.
+//  Copyright © 2021 Gao Sun. All rights reserved.
+//
+
+import SharedLibrary
+import SwiftUI
+
+struct RamProcessRowView: View {
```

Comment:
I think you can re-use `ProcessRowView` and use `valueViewBuilder` for customized value display - would you mind to take a look?